resources/assets/js/api/base.ts

```diff
@@ -1,5 +1,6 @@
 import dayjs from "dayjs";
 import rootAxios from "axios";
+import { string } from "yup";
```

Comment:
Should this be here? Or are you just using the generic string type for typescript?

Reply:
Oops! Nah, just something my IDE added by mistake.